File: teammanual/sample/Sample.java
import java.io.*;
import java.util.*;
public class Sample {
  public static void main(String args[]) throws Exception {
    BufferedReader in = new BufferedReader(new InputStreamReader(System.in));
    int num_testcases = Integer.parseInt(in.readLine());
    for(int i=0;i<num_testcases;++i) {
      StringTokenizer st = new StringTokenizer(in.readLine());
      int a = Integer.parseInt(st.nextToken());
      int b = Integer.parseInt(st.nextToken());
      System.out.println((a+b)+" "+(a-b));
    }
  }
}

File: teammanual/sample/Sample.scala
object Sample {
  def main(args: Array[String]): Unit = {
    val in = new java.io.BufferedReader(
             new java.io.InputStreamReader(System.in));
    val num_testcases = Integer.parseInt(in.readLine());
    for(i <- 1 to num_testcases) {
      val st = new java.util.StringTokenizer(in.readLine());
      val a = Integer.parseInt(st.nextToken());
      val b = Integer.parseInt(st.nextToken());
      println((a+b)+" "+(a-b));
    }
  }
}
